src/app.js

```diff
@@ -27,28 +27,79 @@ export default class App extends React.Component {
 
   }
 
+  homepage(event) {
+    this.setState({page: 'users'}) // Trying to link back to page of users
+  }
+
   render() {
     if(this.state.page == 'login') {
       return(
           <div className = 'container'>
-            <h1>Connect As: </h1>
-            <div className="btn-group" role = "group">
-            <button type="button" className = "btn btn-secondary" id = "tbutton" onClick = { this.connect }>Tourist</button>
-            <button type="button" className = "btn btn-secondary" id = "gbutton" onClick = { this.connect }>Guide</button>
+            <div className="navbar navbar-default">
+              <div className="container-fluid">
+                <div className="navbar-header">
+                  <p id="logo">Wander</p>
+                </div>
+              </div>
+            </div>
+            <h2 id="title">Welcome to Wander! </h2>
+            <div className = 'col-sm-12 col-md-6'>
+              <img src="img/image.png" id="home_img"></img>
+            </div>
+            <div className = 'col-sm-12 col-md-6'>
+              <form>
+                <div className="form-group row">
+                    <i className="fa fa-user" aria-hidden="true"></i>
+                    <input type="text" id = "username_input" className="form-control" placeholder="Username"></input>
+                    <i className="fa fa-key" aria-hidden="true"></i>
```

Comment:
These icons look awkwardly placed relative to the input fields. Either put them on the same line or remove them entirely.